### ISSUE TITLE
Add more language labels to source blocks

### ISSUE THREAD

**Issue by lbolla** [gongzhitaao/orgcss#2]
Source blocks for certain languages get a nice label on the top right of the block, showing which language it is.

The magic seems to happen here:
https://github.com/gongzhitaao/orgcss/blob/1c36ba290677933dd3f58bd8b86a6b43dcd90d9e/src/css/org.scss#L318

Is it possible to add more languages? E.g. Python, Go, Rust, Erlang, Elixir, etc.

**Comment by gongzhitaao** [comment]
Of course.  Just check what CSS class emacs use for certain language when exporting to HTML.  For example, when exporting emacs-lisp, Emacs generates something like

```html
<div class="org-src-container">
  <pre class="src src-emacs-lisp">
  </pre>
</div>
```
Then add 
```css
.org-src-container>pre.src-emacs-lisp:before {
    content: 'whatever name you like';
}
```
I'm just adding a few to demo, you can add more :smile:

**Comment by gongzhitaao** [comment]
Fixed in ad64c12 :clinking_glasses:
